usr/share/rear/layout/save/GNU/Linux/230_filesystem_layout.sh

```diff
@@ -79,6 +79,14 @@ read_filesystems_command="$read_filesystems_command | sort -t ' ' -k 1,1 -u"
             Log "$device is CD/DVD type device [fstype=$fstype], skipping."
             continue
         fi
+        # docker specific exclude part
+        if service docker status >/dev/null 2>&1  ; then
+            # docker daemon/service is running
+            docker_root_dir=$( docker info 2>/dev/null | grep 'Docker Root Dir' | awk '{print $4}' )
+            # if $docker_root_dir is the begining of the $mountpoint then FS is under docker control
+            # and we better exclude from saving the layout
+            echo "$mountpoint" | grep -q "^$docker_root_dir" && continue
```

Comment:
The other automatic hardcoded excludes have a
<pre>
Log "reason why something is excluded"
</pre>
so that I suggest to also do it for automated docker excludes like
<pre>
# if $docker_root_dir is the begining of the $mountpoint then FS is under docker control
# and we better exclude from saving the layout
Log "$device is mounted below $docker_root_dir (mounted under docker control), skipping."
echo "$mountpoint" | grep -q "^$docker_root_dir" && continue
</pre>